scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
+   # Find unity.exe as Start-UnityEditor currently doesn't support arbitrary parameters
+   $editor = Get-ChildItem $UnityPath -Filter 'Unity.exe' -Recurse | Select-Object -First 1 -ExpandProperty FullName
+   
+   # The build output goes to a unique combination of Platform + Arch + Scene to ensure that
+   # each build will have a fresh destination folder.
+   $outDir = "$(Build.ArtifactStagingDirectory)\build\${{ parameters.UnityVersion }}\${{ parameters.Platform }}_${{ parameters.Arch }}_${{ parameters.Scene }}"
```

Comment:
Actually as I look at this a little more, I think there's a bunch of things here that reference YAML things (i.e. ${{ parameters...}, which don't evaluate/have meaning within the context of powershell scripts.

Put another way, you should be able to run this powershell by itself from your local dev box and it should work  (after you pass in explicit parameters to the script) - those explicit parameters should be set on the YAML side.